Create voucher header for mobile

GiovannaValadao opened the pull request:
Create voucher header for mobile:

<img width="374" alt="Captura de Tela 2022-02-09 às 19 58 05" src="https://user-images.githubusercontent.com/42494406/153304674-a72fd46d-c0f0-44e5-921e-eee6d5f16077.png">

GiovannaValadao committed:
create voucher header for mobile

GiovannaValadao committed:
Merge branch 'master' into create-voucher-header-mobile

GiovannaValadao committed:
fix in create voucher header for mobile

raianechristine commented:
the header padding is not equal, I had commented [here](https://github.com/GiovannaValadao/clone-home-ifood/pull/2#discussion_r797682605) to make the spacing equal because there is a greater spacing only at the top and the alignment is not correct: 

![image](https://user-images.githubusercontent.com/38358239/153404599-fdfb865c-523a-4c45-b28e-5f7beaa1762d.png)

if you get confused you can talk to me ❤️

raianechristine commented on src/components/voucherHeaderMobile/VoucherHeaderMobile.jsx:
```diff
@@ -0,0 +1,32 @@
+        </div>
+        <div class="voucher-header-mobile__icon">
+          <span class="voucher-header-mobile__icon-up">
+            <img src={IconUp} />
```
this icon is not visible here,  I think you need to add color to it, or not? 🤔 

<img width="517" alt="image" src="https://user-images.githubusercontent.com/38358239/153405269-ce366dbc-e7d8-40da-a719-ee179b7b2fe4.png">

GiovannaValadao replied:
yes! I have to add the color thanks <3

GiovannaValadao commented:
@raianechristine True, I remember I changed this, but I don't think it was the commit :(

But I changed it now and it looks like this:


<img width="375" alt="Captura de Tela 2022-02-10 às 20 18 20" src="https://user-images.githubusercontent.com/42494406/153513306-098c48b4-3b32-488b-b83a-0c9860b21d66.png">

GiovannaValadao committed:
fix in header and voucherHeader for mobile
